dolphinscheduler-task-plugin/dolphinscheduler-task-datax/src/main/java/org/apache/dolphinscheduler/plugin/task/datax/DataxTask.java

```diff
@@ -94,6 +94,17 @@ public class DataxTask extends AbstractTaskExecutor {
      * datax path
      */
     private static final String DATAX_PATH = "${DATAX_HOME}/bin/datax.py";
+
+    /**
+     * datax hdfs path
+     */
+    private static final String DATAX_HDFS_PATH = "DATAX_HDFS_PATH";
+
+    /**
+     * datax hdfs path
+     */
+    private static final String DATAX_ON_YARN_DEPEND_JAR = "DATAX_ON_YARN_DEPEND_JAR";
```

Comment:
May I ask whether we have an option for users to choose to submit a `DataX` task to yarn or not in UI?

Reply:
The front end is not very good at it. If you can pass a parameter from the front end, you can flexibly judge on the back end

Reply:
Have been added front options, associated pr https://github.com/apache/dolphinscheduler/pull/13937